packages/core/test/integration/model/findAll.test.js

```diff
@@ -1396,6 +1396,34 @@ The following associations are defined on "Worker": "ToDos"`);
         }
       });
 
+      it('should allow a literal as VIRTUAL', async function () {
+        const User = this.sequelize.define('User', {
+          active: {
+            type: Sequelize.VIRTUAL(Sequelize.BOOLEAN, includeAs => [
+              dialectName === 'db2'
+              ? literal(`(SELECT "createdAt" > CURRENT DATE - 7 DAYS FROM "Users" WHERE "User"."id" = "${includeAs.replace(/`/g, '')}"."id")`)
```

Comment:
the table alias should not come pre-escaped. That's what `sql.identifier` is for

Similarly, please don't inline variables in the raw sql. Use the new `sql` tag instead. Here is the documentation about it: https://648dd325deb93b14ad86d83a--sequelize-site.netlify.app/docs/v7/querying/raw-queries/